zmon_aws_agent/postgresql.py

```diff
@@ -8,27 +8,41 @@
 from zmon_aws_agent.aws import entity_id
 from zmon_aws_agent.common import call_and_retry
 
+from opentracing_utils import trace, get_span_from_kwargs
+from opentracing.ext import tags
```

Comment:
I recommend using ``tags as ot_tags`` to avoid confusion when using in functions